roles/mirror_ocp_release/tasks/fetch.yml

```diff
@@ -24,15 +16,4 @@
   until: downloaded is not failed
   when:
     - mor_force or not target.stat.exists
-
-- name: "Figure out status of SELinux"
-  ansible.builtin.command: /usr/sbin/selinuxenabled
-  ignore_errors: true
-  register: _mor_selinux
-  changed_when: false
-
-- name: "Apply new SELinux file context to file"
-  ansible.builtin.command: /usr/sbin/restorecon -R "{{ mor_dir }}/{{ mor_uri | basename }}"
-  become: true
-  when: _mor_selinux.rc == 0
```

Comment:
Restoring the selinux context does not make sense when extracting the artifacts on a temporary directory.
Also, the first tasks in artifacts.yml after including fetch.yml override the context and set it to container_file_t, which should be valid even after moving the artifacts to the target directory served from the cache container.
A different discussion is whether these tasks should be run before or after copying the artifacts to the target directory.